portal/src/main/webapp/js/src/load-frontend.js

```diff
@@ -18,6 +25,7 @@ window.loadReactApp = function(config) {
                 '<div class="alert alert-warning">' +
                 '<button type="button" class="close" data-dismiss="alert">&times;</button>' +
                 'cbioportal-frontend dev mode, using localhost:3000' +
+                '&nbsp;<a href="#" onclick="clearDevState()">clear</a>' +
```

Comment:
I thought about doing this but I think it's a security no-no.  Suppose we let someone designate a heroku instance.  What's to stop a hacker from sending someone a link with the hacker's heroku instance in the url param?  Anyone can make an app on the herokuapp.com domain.  So I think the only safe way to do this would be to have a custom herokuapp domain and limit embeds to that.  Maybe that's possible @inodb.  If we could do what @adamabeshouse suggests, it would make it easier to send around things for review.